src/test/config.sol

```diff
@@ -98,8 +98,8 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x23990e11a1efD07a6000AF0E827A47f2F2086B6E),        // populate with deployed spell if deployed
-            deployed_spell_created:         1666026600,        // use `./scripts/get-created-timestamp.sh <deployment-tx>`
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
+            deployed_spell_created:         0,        // use `./scripts/get-created-timestamp.sh <deployment-tx>`
             previous_spell:                 address(0),        // supply if there is a need to test prior to its cast() function being called on-chain.
             office_hours_enabled:           false,             // true if officehours is expected to be enabled in the spell
```

Comment:
Want to note that we should have Office Hours enabled for mainnet.

Reply:
Noted, will leave spell as is.